test/puppetlabs/puppetdb/middleware_test.clj

```diff
@@ -45,20 +48,26 @@
       (is (= #{"oof/" "rab/" "zab/"} (keyset (@storage :timers))))
       (is (= #{"oof/" "rab/" "zab/"} (keyset (@storage :meters)))))))
 
+(defn create-authorizing-request [hostname]
+  {:scheme :https
+   :ssl-client-cn hostname})
+
 (deftest wrapping-authorization
   (testing "Should only allow authorized requests"
     ;; Setup an app that only lets through odd numbers
-    (let [handler     (fn [req] (-> (rr/response nil)
+    (let [wl (.getAbsolutePath (temp-file "whitelist-log-reject"))
```

Comment:
ick. We should probably plumb this through the regular logger now that we know how to set up separate logs. (I know this is pre-existing)